workflows/cloud-client/src/main/java/com/example/workflows/WorkflowsQuickstart.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.workflows;
+
+// [START workflows_api_quickstart]
+
+// Imports the Google Cloud client library
+
+import com.google.cloud.workflows.executions.v1.Execution;
+import com.google.cloud.workflows.executions.v1.ExecutionsClient;
+import com.google.cloud.workflows.executions.v1.WorkflowName;
+
+/**
+ * Cloud Workflows API sample. Executes a workflow and waits for results.
+ * Example usage:
+ * GOOGLE_CLOUD_PROJECT=myProject \
+ * mvn package exec:java -Dexec.mainClass='com.example.workflows.WorkflowsQuickstart'
+ */
+public class WorkflowsQuickstart {
+  private static final String GOOGLE_CLOUD_PROJECT = System.getenv("GOOGLE_CLOUD_PROJECT");
+  private static String LOCATION = System.getenv("LOCATION");
+  private static String WORKFLOW = System.getenv("WORKFLOW");
+
+  public static String workflowsQuickstart(String projectId, String location, String workflow) {
+    // Execute workflow
+    try (ExecutionsClient workflowExecutionsClient = ExecutionsClient.create()) {
+      WorkflowName parent = WorkflowName.of(projectId, location, workflow);
+      Execution initialExecution = Execution.newBuilder().build();
```

Comment:
Can you add a comment here on what this is doing? It might be more understandable if you use the CreateExecutionRequest.builder